core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -300,8 +300,11 @@ public enum Property {
       "Enables metrics functionality using Micrometer.", "2.1.0"),
   GENERAL_MICROMETER_JVM_METRICS_ENABLED("general.micrometer.jvm.metrics.enabled", "false",
       PropertyType.BOOLEAN, "Enables JVM metrics functionality using Micrometer.", "2.1.0"),
-  GENERAL_MICROMETER_FACTORY("general.micrometer.factory", "", PropertyType.CLASSNAME,
-      "Name of class that implements MeterRegistryFactory.", "2.1.0"),
+  GENERAL_MICROMETER_FACTORY("general.micrometer.factory",
+      "org.apache.accumulo.core.spi.metrics.SimpleLoggingMeterRegistryFactory",
+      PropertyType.CLASSNAMELIST,
+      "A comma separated list of one or more class names that implement MeterRegistryFactory.",
```

Comment:
```suggestion
      "A comma separated list of one or more class names that implement MeterRegistryFactory.  Prior to 2.1.3 the default value was blank.  In 2.1.3 the default value was changed.",
```

Reply:
It would be good to explain the change in expected interface class in the docs and what the support for the old interface class is.

Reply:
Addressed in d8551b56e4